Documentation/contributing.rst

```diff
@@ -729,80 +730,61 @@ illustrating which subset of tests the job runs.
 +-------------------------------------------------------------------------------------+-----------------+-----------------+--------------------+
 | `Cilium-PR-Ginkgo-Tests-All <https://jenkins.cilium.io/job/Cilium-PR-Ginkgo-Tests-All/>`_             | test-all-ginkgo | No                 |
 +-------------------------------------------------------------------------------------------------------+-----------------+--------------------+
-| `Cilium-Pr-Ginkgo-Test-k8s <https://jenkins.cilium.io/job/Cilium-PR-Ginkgo-Tests-K8s/c>`_             | test-missed-k8s | No                 |
+| `Cilium-Pr-Ginkgo-Test-k8s <https://jenkins.cilium.io/job/Cilium-PR-Ginkgo-Tests-k8s/>`_              | test-missed-k8s | No                 |
```

Comment:
Don't we have a script to check for dead links?

Reply:
It doesn't appear to do anything at this time because nowhere in our scripts is `LINKCHECK` enabled.

`tests/98-documentation-link-check.sh `:

```
if [ -z $LINKCHECK ]; then
  echo "Skipping linkcheck, run with LINKCHECK=1 to enable"
  exit 0
fi
```

Reply:
Filed https://github.com/cilium/cilium/issues/3113 to track fixing that.